src/libs/context.js

```diff
@@ -153,6 +160,13 @@ function nameOrFallback () {
   return this.name
 }
 
+/**
+ * determine if an input has a user-defined name
+ */
+function hasGivenName () {
+return typeof this.name !== 'boolean'
```

Comment:
@justin-schroeder this is resulting in the `name` property bleeding through to the radio and checkbox items, which was my experience previously. At some point `this.name` is computed to a string for those items and so this `typeof === 'boolean'` check isn't going to be accurate enough.

Reply:
Tests are good then 👍 Yes, radio and checkbox items forward their context object on down to `FormulateInputGroup`, instead of modifying this globally-applicable rule, we need to purify the data being passed down to radio and checkbox items in `FormulateInputGroup`